src/quo/components/list/item.cljs

```diff
@@ -84,10 +84,12 @@
 
 (defn title-column
   [{:keys [title text-color subtitle subtitle-max-lines
-           title-accessibility-label size text-size title-text-weight]}]
+           title-accessibility-label size text-size title-text-weight
+           right-side-present?]}]
   [rn/view {:style (merge (:tiny spacing/padding-horizontal)
-                          {:justify-content :center
-                           :flex            1})}
+                          ;; make left-side title grow if nothing is present on right-side
+                          (when-not right-side-present? {:flex            1
```

Comment:
I think the map could be on the new line to make lines shorter

Reply:
Okay.